### ISSUE TITLE
Add timed action play

### ISSUE THREAD

**Issue by flinx42** [jrhunger/2600-digits#8]
- Only show each digit [0-9] once on the screen
  allows for more paths to get to the right digit, but also may make it longer to get to it)
-  Shuffle the placement of those 10 digits every XX seconds (could be based on selectable hardness level)
- (optional) have a total play time of XX minutes, that then determines your final score

With this you might only go through the first 10-20 digits, but it be more fun

**Comment by jrhunger** [comment]
True, fun was not really a goal in the original concept, but i like this idea.
